tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -1799,7 +1811,6 @@ def test_with_parms(ansible_zos_module):
 
 def test_with_multiple_of_same_dd_name(ansible_zos_module):
     hosts = ansible_zos_module
-    hosts.all.zos_data_set(name=DEFAULT_DATA_SET, state="absent")
```

Comment:
Was this line (and related) removed because the 'finally' routines should prevent the data set from laying around, or is it replicated somewhere else?

Reply:
Looks like its because its as you say @richp405 , the finally routines are removing all the artifacts so there is no need to perform wasted operations.